scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/TaskKiller.java

```diff
@@ -34,16 +34,19 @@
  */
 package org.ow2.proactive.scheduler.task.utils;
 
+import org.apache.log4j.Logger;
+import org.objectweb.proactive.utils.Sleeper;
+
 public class TaskKiller {
 
-    public enum Status {
-        KILLED_MANUALLY, WALLTIME_REACHED, NOT_YET_KILLED;
-    }
+    private static final Logger logger = Logger.getLogger(TaskKiller.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1312)